navigator_engine/config.py

```diff
@@ -1,15 +1,16 @@
 import logging
-from decouple import config
+import os
+basedir = os.path.abspath(os.path.dirname(__file__))
 
 
 class Config(object):
     DEBUG = False
     TESTING = False
     PRODUCTION = False
-    SQLALCHEMY_DATABASE_URI = 'postgresql://postgres:1234567890@db/engine'
+    SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL') or \
+                              'sqlite:///' + os.path.join(basedir, 'app.db')
```

Comment:
@tomeksabala I'm still trying to work out exactly what you're doing here. Can you please explain? This looks like you are setting the minikube to use SQLLite instead of postrgres... Unless we define a DATABASE_URL env variable somewhere.